package.json

```diff
@@ -167,8 +166,6 @@
     "gm": "^1.23.0",
     "grunt": "~1.0.1",
```

Comment:
Do we still use grunt anywhere else?

Reply:
cmd.sh - there are some notes in the karma file referring to it.

Reply:
Where in the karma file?
I'm just seeing it in cmd.sh as a variable name, that may have meant something before, but now maybe we took out that functionality